docker-compose.yml

```diff
@@ -10,19 +10,14 @@ services:
       - ALTINN_URI=http://wiremock:8080
       - ALTINN_MODEL_PUBLISHER_URI=http://altinn-model-publisher:8080
       - ORGANIZATION_CATALOGUE_URI=https://organization-catalogue.fellesdatakatalog.digdir.no
-      - MONGO_USERNAME=admin
-      - MONGO_PASSWORD=admin
     depends_on:
-      - mongodb
+      - publishers-cache
       - wiremock
 
-  mongodb:
-    image: mongo:latest
-    environment:
-      - MONGO_INITDB_ROOT_USERNAME=admin
-      - MONGO_INITDB_ROOT_PASSWORD=admin
+  publishers-cache:
```

Comment:
kanskje kalle det bare `redis`?

Reply:
Vi endte opp med egne instanser av redis for altinn-model-publisher og fdk-model-publisher, pod-navn i gcp er hhv publishers-cache og fdk-publishers-cache. Bruker like så greit samme navn her, så må vi ikke endre det i config for lokal utvikling vs deploy